runtime/lua/vim/lsp/handlers.lua

```diff
@@ -6,14 +6,28 @@ local api = vim.api
 local buf = require 'vim.lsp.buf'
 
 local M = {}
+local disabled_errors = {}
```

Comment:
should be configurable maybe?

Reply:
This is just what I'm using to cache disabled errors, but on that topic most of what has been done this far uses overriding handlers, do you have a preference/should we add a section in global options for dealing with language server settings?